src/web3/networkConfig.ts

```diff
@@ -482,26 +483,13 @@ export const networkIdToWalletConnectChainId: Record<NetworkId, string> = {
   [NetworkId['op-sepolia']]: 'eip155:11155420',
 }
 
-export const walletConnectChainIdToNetworkId: Record<string, NetworkId> = {
-  'eip155:44787': NetworkId['celo-alfajores'],
-  'eip155:42220': NetworkId['celo-mainnet'],
-  'eip155:1': NetworkId['ethereum-mainnet'],
-  'eip155:11155111': NetworkId['ethereum-sepolia'],
-  'eip155:42161': NetworkId['arbitrum-one'],
-  'eip155:421614': NetworkId['arbitrum-sepolia'],
-  'eip155:10': NetworkId['op-mainnet'],
-  'eip155:11155420': NetworkId['op-sepolia'],
-}
+export const walletConnectChainIdToNetworkId: Record<string, NetworkId> = _.invert(
+  networkIdToWalletConnectChainId
+) as Record<string, NetworkId>
 
-export const walletConnectChainIdToNetwork: Record<string, Network> = {
-  'eip155:44787': Network.Celo,
-  'eip155:42220': Network.Celo,
-  'eip155:1': Network.Ethereum,
-  'eip155:11155111': Network.Ethereum,
-  'eip155:42161': Network.Arbitrum,
-  'eip155:421614': Network.Arbitrum,
-  'eip155:10': Network.Optimism,
-  'eip155:11155420': Network.Optimism,
+export const walletConnectChainIdToNetwork: Record<string, Network> = {}
```

Comment:
I spent a sec trying to code golf this into a single line using a `Object.entries(walletConnectChainIdToNetworkId).reduce(...)`, but then realized that it's prolly less readable than the current approach :laughing:

Reply:
I think you could do `Object.fromEntries(Object.entries(walletConnectChainIdToNetworkId).map(([chainId, networkId]) => [networkId, chainId])))` . But i went with the for loop because it seemed more readable

Reply:
i may have gotten network id and chain id backwards but whatever